ebb_lint/test/test_flake8.py

```diff
@@ -1,13 +1,15 @@
+# -*- coding: utf-8; -*-
 from __future__ import unicode_literals
 
 import ast
 import functools
+import pytest
 import re
+import six
 import sys
 
-import pytest
-import six
 from flake8.engine import get_parser
+from io import BytesIO
```

Comment:
I don't quite understand this import reordering. `pytest` and `six` are third-party modules, and `io` is stdlib.